converter/internal/staticconvert/staticconvert.go

```diff
@@ -129,6 +129,12 @@ func appendStaticPromtail(f *builder.File, staticConfig *config.Config) diag.Dia
 		promtailConfig.TargetConfig = logConfig.TargetConfig
 		promtailConfig.LimitsConfig = logConfig.LimitsConfig
 
+		// We are using the
+		err := promtailConfig.ServerConfig.Config.LogLevel.Set("info")
+		if err != nil {
+			panic("unable to default promtail log level from the static converter.")
```

Comment:
verbiage seems off here, "unable to set default promtail..." instead?

Reply:
good call, updated